frontend/packages/react-app/src/components/templates/TokenInformationTemplate/index.tsx

```diff
@@ -12,11 +14,18 @@ export interface TokenInformationTemplateProps {
 const TokenInformationTemplate: React.FC<TokenInformationTemplateProps> = ({
   state,
   dispatch,
-}) => (
-  <div style={{ minHeight: "100vh" }}>
-    <AppHeader />
-    <TokenInformationTabs state={state} dispatch={dispatch} />
-  </div>
-);
+}) => {
+  const { active } = useWeb3React();
```

Comment:
Because of this props, storybook show no content. 
Could you move this to page component like following? 
https://github.com/TART-tokyo/iroiro/blob/e5cefab7d725b518352e9186524921e803021da5/frontend/packages/react-app/src/components/templates/CreateCampaignPageTemaplate/index.tsx#L17

Reply:
Fix